# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/moz.build
# Commit: 8f0534e7ae1f
# Full Hash: 8f0534e7ae1fb3c7dba19afb67d5c3aaf20f1a74
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Fix VideoConduit unittests. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D105777
# ==============================================================================

diff -r 65c1552a1b96 -r 8f0534e7ae1f media/webrtc/signaling/gtest/moz.build
--- a/media/webrtc/signaling/gtest/moz.build	Fri Feb 19 13:48:09 2021 +0100
+++ b/media/webrtc/signaling/gtest/moz.build	Fri Feb 19 13:53:22 2021 +0100
@@ -44,7 +44,7 @@
         "jsep_track_unittest.cpp",
         "mediapipeline_unittest.cpp",
         "sdp_unittests.cpp",
-#        "videoconduit_unittests.cpp",
+        "videoconduit_unittests.cpp",
     ]
 
     FINAL_LIBRARY = "xul-gtest"